### DISCUSSION TITLE
Any plans to support Kubernetes pods?

### DISCUSSION THREAD

**Discussion by sinkr** [amitds1997/remote-nvim.nvim#106] [Q&A]
I don't feel like `kubectl exec` is too far different from `docker exec`, but I may not be understanding how the plugin will work.

Thanks.

**Comment by amitds1997** [comment]
You are quite correct. This would in fact be easier than `docker exec` since it allows one to do port forwarding with a command. This is something which is not present in docker :/

The implementation would be very similar to what I did in https://github.com/amitds1997/remote-nvim.nvim/pull/58 and should not be hard to implement. I would add it to the backlog.

**Reply by amitds1997** [reply]
Created https://github.com/amitds1997/remote-nvim.nvim/issues/110
